src/components/login-form.js

```diff
@@ -1,8 +1,7 @@
 import PropTypes from 'prop-types';
```

Comment:
Both the code here and the UI look alot better now 👍 
Maybe we could make a ticket to update registration in the same direction.

Reply:
Yes, I created a ticket: https://educandu.atlassian.net/browse/EDU-462